gax/src/fallback.ts

```diff
@@ -100,6 +100,7 @@ export class GrpcClient {
   private static protoCache = new Map<string, protobuf.Root>();
   httpRules?: Array<google.api.IHttpRule>;
   numericEnums: boolean;
+  minifyJson: boolean;
```

Comment:
Should this be optional? `minifyJson?`

Reply:
This is [handled in the constructor where it converts an unset value to default value](https://github.com/googleapis/gax-nodejs/blob/47417e4d97a43b4cbb2c4ac16e574201b65edc43/gax/src/fallback.ts#L147), and follows the existing pattern of the optional param `numericEnums`.  I'm still happy to change it to optional if you prefer that.